pom.xml

```diff
@@ -1416,58 +1304,84 @@
+                                <exclude>**/TypeHelperTest.java</exclude>
+                                <exclude>**/ConcurrentReferenceHashMap.java</exclude>
+                            </excludes>
+                            <indent>
+                                <tabs>true</tabs>
+                                <spacesPerTab>4</spacesPerTab>
+                            </indent>
+
+                            <importOrder>
+                                <!-- you can use an empty string for all the imports you didn't specify explicitly, '|' to join group without blank line, and '\#` prefix for static imports. -->
+                                <order>\#,java,javax,jakarta,org.openjdk.jmh,org.hibernate,org.hibernate.testing,org.hibernate.test,org.junit,org.jboss,com,javafx.beans,</order>
+                            </importOrder>
+                            <trimTrailingWhitespace/>
+                            <endWithNewline/>
+                            <eclipse>
+                                <!-- optional version of Eclipse Formatter -->
+                                <!-- <version>4.26</version> -->
+                                <file>hibernate_validator_style.xml</file>
+                            </eclipse>
+                            <removeUnusedImports>
+                                <!-- default one fails to process the classes -->
+                                <engine>cleanthat-javaparser-unnecessaryimport</engine>
+                            </removeUnusedImports>
+                            <licenseHeader>
+                                <file>license.java.header</file>
+                                <!-- Delimiter is a regex which identifies the "top" of the file i.e. all above it is considered a header -->
+                                <delimiter>(\n)*^(\/\/tag.+|\/\/spotless.+|package .+;|//.+)$</delimiter>
+                            </licenseHeader>
+                            <replaceRegex>
+                                <name>Remove wildcard imports</name>
+                                <searchRegex>^import\s+[^\*\s]+\*;$</searchRegex>
+                                <replacement>$1</replacement>
+                            </replaceRegex>
+                        </java>
                     </configuration>
-                    <executions>
-                        <execution>
-                            <id>import-sorting</id>
-                            <goals>
-                                <goal>${goal.impsort-maven-plugin}</goal>
-                            </goals>
-                            <phase>process-sources</phase>
-                        </execution>
-                    </executions>
-                </plugin>
-                <plugin>
-                    <groupId>net.revelc.code.formatter</groupId>
-                    <artifactId>formatter-maven-plugin</artifactId>
-                    <version>${version.formatter-maven-plugin}</version>
```

Comment:
So we *can* replace both of these plugins with just spotless? That's already a nice improvement, no?